test/custom_operator/test_custom_ops.cpp

```diff
@@ -43,6 +43,51 @@ void get_operator_from_registry_and_execute() {
   }
 }
 
+void get_autograd_operator_from_registry_and_execute() {
+  auto& ops = torch::jit::getAllOperatorsFor(
+      torch::jit::Symbol::fromQualString("custom::op_with_autograd"));
+  AT_ASSERT(ops.size() == 1);
+
+  auto& op = ops.front();
+  AT_ASSERT(op->schema().name() == "custom::op_with_autograd");
+
+  torch::jit::Stack stack;
+  torch::Tensor x = torch::randn({5,5}, torch::requires_grad());
+  torch::Tensor y = torch::randn({5,5}, torch::requires_grad());
+  torch::jit::push(stack, x, 2, y);
+  op->getOperation()(stack);
+  torch::Tensor output = torch::jit::pop(stack).toTensor();
+  AT_ASSERT(0 == stack.size());
```

Comment:
This is pretty boilerplatey, I think there's a simple utility function you could write to reduce the boilerplate quite a bit.

Reply:
Added a new PR https://github.com/pytorch/pytorch/pull/31282 to also clean this up for pre-existing unit tests.